app/assets/templates/graph_template.html.erb

```diff
@@ -51,16 +51,33 @@
           <label>Graph Title:</label>
           <input class="form-control" ng-model="graph.title">
           <br/>
-          <div class="col-xs-12">
-            <label>Interpolation:</label>
-            <div class="radio">
-              <label class="graph_settings_image cardinal"><input type="radio" ng-model="graph.interpolationMethod" value="cardinal"> cardinal</label>
-            </div>
-            <div class="radio">
-              <label class="graph_settings_image linear"><input type="radio" ng-model="graph.interpolationMethod" value="linear"> linear</label>
+          <div class="row">
+            <div class="col-xs-6">
+              <label>Interpolation:</label>
+              <div class="radio">
+                <label class="graph_settings_image cardinal"><input type="radio" ng-model="graph.interpolationMethod" value="cardinal"> cardinal</label>
+              </div>
+              <div class="radio">
+                <label class="graph_settings_image linear"><input type="radio" ng-model="graph.interpolationMethod" value="linear"> linear</label>
+              </div>
+              <div class="radio">
+                <label class="graph_settings_image step"><input type="radio" ng-model="graph.interpolationMethod" value="step-after"> step</label>
+              </div>
             </div>
-            <div class="radio">
-              <label class="graph_settings_image step"><input type="radio" ng-model="graph.interpolationMethod" value="step-after"> step</label>
+            <div class="col-xs-6">
+              <label>Resolution:
+                <div class="resolution">
+                  <div ui-slider="" min="1" max="3" ng-model="graph.resolution"
+                    class="ui-slider ui-slider-horizontal ui-widget ui-widget-content ui-corner-all" aria-disabled="false">
+                    <a class="ui-slider-handle ui-state-default ui-corner-all" href="#" style="left: 0%;"></a>
+                  </div>
+                  <ul class="resolution_setting_description">
+                    <li>low</li>
```

Comment:
Could we have 5 states? Maybe lowest, low, medium, high, highest.

I'd suggest making "high" as similar as possible to today's default resolution (since today's default resolution is pretty high), and defaulting to high resolution. As for calibration of the high and low ends of the scale, I think the current ends are already pretty good, but could be made a tiny tad more extreme (lowest would be a tiny bit lower than your current low setting, highest would be a tiny bit higher than your current high setting - everything else would be spread evenly between those extremes).